src/libraries/Common/tests/System/Xml/XmlDiff/XmlDiffDocument.cs

```diff
@@ -1550,15 +1560,19 @@ public class XmlDiffCharacterData : XmlDiffNode
     {
         private string _value;
         private XmlDiffNodeType _nodetype;
-        public XmlDiffCharacterData(string value, XmlDiffNodeType nt, bool NormalizeNewline)
+        public XmlDiffCharacterData(string value, XmlDiffNodeType nt, bool NormalizeNewline, bool NormalizeSpaces, bool IgnoreWhitespace)
```

Comment:
> NormalizeSpaces, bool IgnoreWhitespace [](http://example.com/codeflow?start=97&length=39)

it is weird we are using parameters started with upper case letter!

Reply:
Changed, was just following the pattern.